### PR TITLE
Update html5.markdown

### PR DESCRIPTION
**Pull request by energywave**
## Proposed change

Clarified what mail was needed in vapid_email as it recalls "see configuring the platform" and there, at point 9, was suggested to write down the mail you can find in service account tab, Firebase service account field it was implicit that the mail was that. But it's not! You have to insert your google account mail that's associated with the firebase account. Hope that helps people to not struggle like me trying ot understand why it's not working...

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @energywave, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17694).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/html5.markdown [hunk 1]
```diff
@@ -65,7 +65,7 @@ vapid_prv_key:
   required: true
   type: string
 vapid_email:
-  description: The e-mail account associated with your Firebase project, [see configuring the platform](#configuring-the-platform).
+  description: The e-mail account of your google account associated with your Firebase project, [see configuring the platform](#configuring-the-platform).
```
Comment (frenck) [inline source/_integrations/html5.markdown]:
```suggestion
  description: The e-mail account of your Google account associated with your Firebase project, [see configuring the platform](#configuring-the-platform).
```

File: source/_integrations/html5.markdown [hunk 2]
```diff
@@ -99,7 +99,7 @@ The `html5` platform can only function if all of the following requirements are
 6. Select the ['Cloud Messaging' tab](https://console.firebase.google.com/project/_/settings/cloudmessaging).
 7. Generate a new key pair under the Web configuration listing at the bottom of the page. To view the private key click the three dots to the right and 'Show private key'.
 8. Select the ['Service Accounts' tab](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk).
-9. Get the email address for the project under the text that says "Firebase service account".
+9. Get the email address for the project under the text that says "Firebase service account" for reference. (This is NOT what you have to insert in vapid_email! You'll have to insert your google account e-mail there.)
```
Comment (frenck) [inline source/_integrations/html5.markdown]:
```suggestion
9. Get the email address for the project under the text that says "Firebase service account" for reference. This is not what you have to insert in `vapid_email`! You'll have to insert your Google account e-mail there.
```

**Review by frenck — APPROVED** [review]
Thanks, @energywave 👍

**Pull request merged**